src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -125,14 +125,6 @@ class ConsignmentService(
     consignmentRepository.consignmentHasFiles(consignmentId)
   }
 
-  def getConsignmentFileProgress(consignmentId: UUID): Future[FileChecks] = {
```

Comment:
I think the following functions can now be deleted, as they don't appear to be used anywhere expect in their own unit tests, and the unit tests can be removed as well:

- `def countProcessedAvMetadataInConsignment(consignmentId: UUID): Future[Int]`
- `def countProcessedChecksumInConsignment(consignmentId: UUID): Future[Int]`
- `def countProcessedFfidMetadata(consignmentId: UUID): Future[Int]`

Reply:
I can't believe I missed a chance to go deleting large parts of the API code. I've removed them and their associated tests.